src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -149,6 +140,15 @@
               <text value="Replace the Run and Debug console output with an experimental Flutter Logging view"/>
             </properties>
           </component>
+          <component id="33088" class="javax.swing.JCheckBox" binding="mySyncAndroidLibrariesCheckBox">
+            <constraints>
+              <grid row="3" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Show Android libraries; for working with Java/Kotlin (requires reopening projects)"/>
```

Comment:
'Show Android libraries; for working with Java/Kotlin (requires reopening projects)'

Perhaps, 'When opening a Flutter project, run gradle to discover plugin native libraries'.?

The tooltip could then indicate why you'd want to  (for code completion and such).